src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/BaseTests.cs

```diff
@@ -24,18 +24,20 @@ static BaseTests()
 
         protected IFaceAPI GetFaceClient(DelegatingHandler handler)
         {
-            IFaceAPI client;
-            client = new FaceAPI(new ApiKeyServiceClientCredentials(FaceSubscriptionKey), handlers: handler);
-            client.AzureRegion = Face.AzureRegions.Westcentralus;
+            IFaceAPI client = new FaceAPI(new ApiKeyServiceClientCredentials(FaceSubscriptionKey), handlers: handler)
+            {
+                AzureRegion = Face.AzureRegions.Westcentralus
+            };
 
             return client;
         }
 
         protected IComputerVisionAPI GetComputerVisionClient(DelegatingHandler handler)
         {
-            IComputerVisionAPI client;
-            client = new ComputerVisionAPI(new ApiKeyServiceClientCredentials(ComputerVisionSubscriptionKey), handlers: handler);
-            client.AzureRegion = Vision.AzureRegions.Westcentralus;
+            IComputerVisionAPI client = new ComputerVisionAPI(new ApiKeyServiceClientCredentials(ComputerVisionSubscriptionKey), handlers: handler)
```

Comment:
Better to split the test for ComputerVision and Face, I see no gain to merge them together.
Two teams working on the same file will bring high potential of conflicts.

Reply:
Done